src/elf.rs

```diff
@@ -2,7 +2,8 @@
 
 use goblin;
 use failure::Error;
-use {artifact, Artifact, Decl, Object, Target, Ctx, ImportKind, RelocOverride};
+use {artifact, Artifact, Decl, Object, Target, Ctx, ImportKind};
+pub use artifact::LinkAndDecl;
```

Comment:
What's the reason for a "pub use" outside of a lib.rs or mod.rs file?

Reply:
it was a mistake :) ill push a fix and then merge it